src/main/scala/io/citrine/lolo/transformers/FeatureRotator.scala

```diff
@@ -85,6 +85,10 @@ class RotatedFeatureTrainingResult(
   override def getModel(): Model[PredictionResult[Any]] = {
     new RotatedFeatureModel(baseTrainingResult.getModel(), rotatedFeatures, trans)
   }
+
+  override def getLoss(): Option[Double] = baseTrainingResult.getLoss()
+
+  override def getPredictedVsActual(): Option[Seq[(Vector[Any], Any, Any)]] = baseTrainingResult.getPredictedVsActual()
```

Comment:
I think you need to un-rotate the feature vectors here.

Reply:
You're right. Fixed.